test/debug.jl

```diff
@@ -427,7 +426,12 @@ end
 
         frame = JuliaInterpreter.enter_call(sort, a)
         frame = stepkw!(frame)
-        @test frame.pc == JuliaInterpreter.nstatements(frame.framecode) - 1
+        @static if VERSION ≥ v"1.7"
+            # TODO fix this broken test (@aviatesk)
```

Comment:
This test case is broken on `≥v"1.11-"`. We need to revisit the implementation of `maybe_step_through_kwprep!`
https://github.com/JuliaDebug/JuliaInterpreter.jl/blob/eae07f9c6f8155f2d1edb7638834856da732f466/src/commands.jl#L273-L278